examples/plotting/xy_lims.py

```diff
@@ -41,7 +41,7 @@
 # determine which pixel coordinates these represent on the Map.
 
 world_coords = SkyCoord(Tx=xlims_world, Ty=ylims_world, frame=aia_map.coordinate_frame)
-pixel_coords = aia_map.world_to_pixel(world_coords)
+pixel_coords = aia_map.wcs.world_to_pixel(world_coords)
 
 # we can then pull out the x and y values of these limits.
 xlims_pixel = pixel_coords.x.value
```

Comment:
This example should also fail if I understand correctly.

Reply:
> ill check on this . also why does changelog needs to be reset. It should show that we have changed to wcs right

That is not how we do that. The file should be changed back to normal.

Reply:
Sure, but now we don't have a pixel pair, you have to change the lines in th example that assume it is a pixel pair.

Reply:
No, but we don't need that. We can change the other lines in the example.

Reply:
We should not be using PixelPair, you need to change the other lines of the example.

The changelog file still has changes.